soccer/gameplay/plays/testing/four_courners_pass.py

```diff
@@ -0,0 +1,229 @@
+import robocup
+import play
+import behavior
+import skills.move
+import skills.capture
+import tactics.coordinated_pass
+import constants
+import main
+import enum
+
+
+## A testing play to demonstrate our ability to pass and recieve balls
+# One robot will pursue the ball while three other robots will pass the ball amongst themselves
+# they can only pass the ball at the corners and recieve at the corners and they cannot move
+# through the center of the square
+class FourCornerPass(play.Play):
+	class State(enum.Enum):
+		## One robot goes captures the ball while the other two gets on corners
+		# pursuing robot does not move on the first instance
+		setup = 1
+
+		## The robots pass to each other and begin setting up the next pass
+		# while the chasing robot goes for the ball.
+		passing = 2
+
+	def __init__(self):
+		super().__init__(continuous=True)
+
+		# register states - they're both substates of "running"
+		self.add_state(FourCornerPass.State.setup,
+					   behavior.Behavior.State.running)
+		self.add_state(FourCornerPass.State.passing,
+					   behavior.Behavior.State.running)
```

Comment:
This can be a for loop

Reply:
But it's only like two lines. I can if it's really needed

Reply:
Nah, it's just for future reference